src/templates/home-page/HarnessThePower/ImageStack.tsx

```diff
@@ -24,7 +24,11 @@ const ImageStack: FC<{
           <Square bg="brand.500" w="20px" h="20px" />
           <LabelMd>{title}</LabelMd>
         </HStack>
-        <BodyMd mt="1rem !important">{description}</BodyMd>
+        <Box
```

Comment:
I don't like losing our typography components. What if we just updated the components themselves instead of changing each use to a `<Box>` component?

something like this?

```
export const BodyMd: FC<TextProps> = (props) => {
  return <Box as="p" layerStyle="bodyMd" {...props} />
}
```

Reply:
this could work! 👍  Mostly I just want to preserve our typography and styling in a component, instead of passing layer/text styles each time.

Reply:
Done [2f991de](https://github.com/threshold-network/splash-page/pull/22/commits/2f991de3e19d390853ac27ca127a62a04e0c292b).